ios/Sources/AboutFeature/AboutDroidKaigiScreen.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import Styleguide
+
+public struct AboutDroidKaigiScreen: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    public init() {
+        UITableView.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().barTintColor = .clear
+    }
+
+    public var body: some View {
+        NavigationView {
+            VStack {
+                VStack {
+                    Text(L10n.AboutDroidKaigiScreen.whatIs).font(.system(size: 22))
+                    AssetImage.logoTitle.image
+                        .frame(width: 261.81, height: 41.06, alignment: .center)
```

Comment:
Frame size should be 262 x 44
https://www.figma.com/file/D2VLqh0xOXbH0zB6cTz053/DroidKaigi_2021_official_App-iOS?node-id=159%3A4315

Reply:
DONE

Reply:
Please set frame  aspect ratio 262 x 44 and sure it fits design. 🙏🏼  (Current image size is too small)

These modifiers ( `resizable` , `aspectRatio` ) are useful I think.